docs/src/components/component/button-trigger/button-trigger.config.yml

```diff
@@ -0,0 +1,129 @@
+title: button
+status: wip
+collated: true
+variants:
+  - name: default
+    context:
+      isDefault: true
+      buttonStatus: primary
+      buttonText: default
+  - name: base
+    context:
+      isDefault: true
+      buttonStatus: base
+      buttonText: base
+  - name: warning
+    context:
+      isDefault: true
+      buttonStatus: warning
+      buttonText: warning
+  - name: active
+    context:
+      isDefault: true
+      buttonStatus: primary
+      buttonText: active
+  - name: disabled
+    context:
+      isDefault: true
+      buttonStatus: cautious
+      buttonText: disabled
+  - name: finish
+    context:
+      isDefault: true
+      buttonStatus: finish
+      buttonText: finish-outline
+  - name: base-outline
+    context:
+      isOutline: true
+      buttonStatus: base
+      buttonStyle: outline
+      buttonText: base-outline
+  - name: warning-outline
+    context:
+      isOutline: true
+      buttonStatus: warning
+      buttonStyle: outline
+      buttonText: warning-outline
+  - name: active-outline
+    context:
+      isOutline: true
+      buttonStatus: primary
+      buttonStyle: outline
+      buttonText: active-outline
+  - name: disabled-outline
+    context:
+      isOutline: true
+      buttonStatus: cautious
+      buttonStyle: outline
+      buttonText: disabled-outline
+  - name: finish-outline
+    context:
+      isOutline: true
+      buttonStatus: finish
+      buttonStyle: outline
+      buttonText: finish-outline
+  - name: base-open
+    context:
+      isDefault: true
+      isOpen: true
+      buttonStatus: base
+      buttonText: base
+  - name: warning-open
+    context:
+      isDefault: true
+      isOpen: true
+      buttonStatus: warning
+      buttonText: warning
+  - name: active-open
+    context:
+      isDefault: true
+      isOpen: true
+      buttonStatus: primary
+      buttonText: active
+  - name: disabled-open
+    context:
+      isDefault: true
+      isOpen: true
+      buttonStatus: cautious
+      buttonText: disabled
+  - name: finish-open
+    context:
+      isDefault: true
+      isOpen: true
+      buttonStatus: finish
+      buttonText: finish-outline
+  - name: base-outline-open
+    context:
+      isOutline: true
+      isOpen: true
+      buttonStatus: base
+      buttonStyle: outline
+      buttonText: base-outline
+  - name: warning-outline-open
+    context:
+      isOutline: true
+      isOpen: true
+      buttonStatus: warning
+      buttonStyle: outline
+      buttonText: warning-outline
+  - name: active-outline-open
+    context:
+      isOutline: true
+      isOpen: true
+      buttonStatus: primary
+      buttonStyle: outline
+      buttonText: active-outline
+  - name: disabled-outline-open
+    context:
+      isOutline: true
+      isOpen: true
+      buttonStatus: cautious
+      buttonStyle: outline
+      buttonText: disabled-outline
+  - name: finish-outline-open
+    context:
+      isOutline: true
```

Comment:
Yeah, this is part of a larger question about how important consistency is in the docs and how closely the PL patterns can/should match the production React patterns.

I added the boolean values here (like `isWhatever`) before I upgraded to the nunjucks template — handlebars couldn't handle "complex" logic like == matching.